src/main/java/servlets/PlacesListServlet.java

```diff
@@ -24,7 +24,7 @@
     @Override
     public void init() {
         this.context = new GeoApiContext.Builder()
-            .apiKey("API_KEY_HERE")
+            .apiKey("AIzaSyCN-bDjEE_TtP8Md5Mm7RIdhpc6v9h3QoQ")
```

Comment:
Do avoid commiting API keys for security reasons, especially for the server side key where there are no restrictions on its usage